vendor-extra/LiberoContentBundle/tests/ViewConverter/ItalicVisitorTest.php

```diff
@@ -37,8 +37,8 @@ public function it_does_nothing_if_it_is_not_a_libero_italic_element(string $xml
 
     public function nodeProvider() : iterable
     {
-        yield 'different namespace' => ['<i xmlns="http://example.com">foo</i>'];
-        yield 'different element' => ['<b xmlns="http://libero.pub">foo</b>'];
+        yield 'different namespace' => ['<italic xmlns="http://example.com">foo</italic>'];
+        yield 'different element' => ['<bold xmlns="http://libero.pub">foo</bold>'];
```

Comment:
Decided to change these also in anticipation on support for `libero:bold`. See: https://github.com/libero/schemas/pull/15